### ISSUE TITLE
Middle click isn't working.

### ISSUE THREAD

**Issue by Pauan** [Pauan/tab-organizer#50]
Originally reported on Google Code with ID 50

```
What steps will reproduce the problem?
1. Open Tab Organizer.
2. Middle click on a tab.

What is the expected output? What do you see instead?
Tab should be closed. Instead, nothing happens.

Please provide any additional information below.
Did this regress from 2.7? Why? I haven't changed the tab clicking code, so this is
pretty weird.
```

Reported by `pcxunlimited` on 2010-09-17 16:15:51

**Comment by Pauan** [comment]
```
It's not a problem with my code. Something broke in Chrome. I could change it to use
"mouseup", except that works differently from "click", so I'd rather not do that.
```

Reported by `pcxunlimited` on 2010-09-18 16:09:20
- **Status changed**: `ExternalDependency`
- **Labels removed**: Milestone-2.9
- **Blocked on**: #56124

**Comment by Pauan** [comment]
Reported by `pcxunlimited` on 2010-09-18 17:14:34
- **Labels added**: Priority-High
- **Labels removed**: Priority-Medium

**Comment by Pauan** [comment]
```
Looks like this won't be fixed on the Chrome end, and even if it is, I can expect it
to take a while. In the meantime, I'll use "mouseup".
```

Reported by `pcxunlimited` on 2010-09-18 21:41:26
- **Status changed**: `Delegated`
- **Labels added**: Milestone-2.9

**Comment by Pauan** [comment]
```
This issue was updated by revision 884f12822e.
```

Reported by `pcxunlimited` on 2010-09-18 21:46:07
- **Status changed**: `Pending`

**Comment by Pauan** [comment]
Reported by `pcxunlimited` on 2010-09-19 18:02:54
- **Status changed**: `Verified`

**Comment by Pauan** [comment]
Reported by `pcxunlimited` on 2012-08-27 23:44:47
- **Status changed**: `Fixed`
